storage/badger/operation/prefix.go

```diff
@@ -56,8 +56,8 @@ const (
 	codeBlockIDToQuorumCertificate = 45 // index of quorum certificates by block ID
 
 	// codes for indexing multiple identifiers by identifier
-	// NOTE: 51 was used for identity indexes before epochs
 	codeBlockChildren     = 50 // index mapping block ID to children blocks
+	_                     = 51 // DEPRECATED: 51 was used for identity indexes before epochs
```

Comment:
I would be inclined to reassign the prefix `51 `to `codeAllBlockReceipts`, which currently uses value `59`:
https://github.com/onflow/flow-go/blob/54ea60dc927f204f42fc8b7dc58fb118f53f188c/storage/badger/operation/prefix.go#L68

toughts?